### ISSUE TITLE
gptel-directives set variable

### ISSUE THREAD

**Issue by silgon** [karthink/gptel#411]
Sorry for the seamless silly question. I'm trying to set the directives:
![image](https://github.com/user-attachments/assets/8eaec518-725b-415e-94c2-eeb933913944)
I tried to do it simply with:
```elisp
(setq gptel-directives
    '((default . "You are a large language model living in Emacs and a helpful assistant. Respond concisely.")
         (programming . "You are a large language model and a careful programmer. Provide code and only code as output without any additional text, prompt, note or ```.")
         (writing . "You are a large language model and a writing assistant. Respond concisely.")
         (chat . "You are a large language model and a conversation partner. Respond concisely.")))
```
But it doesn't seem to do the job. I tried a bunch of other things too (with add-hook 'gptel-mode-hook, with custom-set-variables , etc) but without success. How can I set my own directives?

**Comment by karthink** [comment]
From your screenshot it looks like it worked?  How is it failing?

**Comment by silgon** [comment]
gpt4o-mini is adding me always triple quote when using it as a programmer, I just included ``` for the example. However, I would like to have more and different directives also.
Sorry for the subtle change 😅

**Comment by karthink** [comment]
Setting `gptel-directives` as you do should work correctly.  I'm not sure what could be going wrong.  Did you use the customize interface to set its value?

**Comment by silgon** [comment]
I have absolutely not idea what a customized interface is. I just set the line I posted to my .emacs  (actually, .emacs.d/init.el) file =)

**Comment by silgon** [comment]
Wow.... it's crazy, i reopened emacs I don't know how many times and nothing happened. After your reply, I just restarted again to test (and checking that I'm not becoming crazy), and well.... turns out it worked 😓. Anyway, not sure what happened, but well, programming is like that right? After n number of years you just accept it =P
BTW, I saw yesterday the video for this library and the other for latex. You rock men! 😉 (and it also reminded me of the time I was doing my Ph.D.). Cheers!
